python-sdk/example_dags/example_load_file.py

```diff
@@ -34,7 +33,7 @@
 with dag:
     # [START load_file_example_1]
     my_homes_table = aql.load_file(
-        input_file=File(path="s3://astro-sdk/sample.csv"),
+        input_file=File(path="s3://astro-sdk/python_sdk/example_dags/data/sample.csv"),
```

Comment:
Perhaps we can `s3://astro-sdk/python_sdk/example_dags/data` as a prefix constant

Reply:
Created a issue for this https://github.com/astronomer/astro-sdk/issues/1299